go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/Prnyself/xmlrpc
```

Comment:
This should be `github.com/kolo/xmlrpc`

Reply:
whoops, sorry I just open this PR from my own project :(